config/schemas/6/rapidast_schema.json

```diff
@@ -639,25 +639,16 @@
                         "type": "object",
                         "description": "Garak Scanner",
                         "properties": {
-                            "model_name": {
-                                "type": "string"
-                            },
-                            "model_type": {
-                                "type": "string"
-                            },
-                            "generators": {
+                            "parameters": {
```

Comment:
Since `model_type` is a required field, wouldn't it make more sense to define it here rather than checking for its existence in the code? After all, that's what the schema file is for
```
"parameters": {
    "type": "object",
    "properties": {
        "plugins": {
            "type": "object",
            "properties": {
                "type": "object",
                "properties": {
                    "model_type": {
                        "type": "string"
                    }
                }
            },
            "required": ["model_type"]
        }
    },
    "required": ["plugins"]
},
```

Reply:
will address this too in a separate PR.